src/core/qgsproject.cpp

```diff
@@ -511,6 +511,7 @@ void QgsProject::clear()
   mCustomVariables.clear();
   mMetadata = QgsProjectMetadata();
   mMetadata.setCreationDateTime( QDateTime::currentDateTime() );
+  mMetadata.setAuthor( QgsApplication::userFullName() );
```

Comment:
I'm -0 on this -- working in a sensitive environment, and working with other people in the same environment, I cannot recall the number of times someone had his/her name (or a user name that's easily linked to the person) attached to a word document or PDF without his/her knowledge.

I recon this isn't necessarily the most common scenario, just serving food for thoughts :)

Reply:
How about a hidden settings key which avoids this?

Reply:
@nyalldawson , sounds like a reasonable compromise.

Reply:
Done